Formula/weaver.rb

```diff
@@ -0,0 +1,36 @@
+class Weaver < Formula
+  desc "Command-line tool for Weaver"
+  homepage "https://github.com/scribd/Weaver"
+  url "https://github.com/scribd/Weaver/archive/0.10.0.tar.gz"
+  sha256 "93dee436fcb13e60d379393da2e7d734c44037d95e97f2917982351446c81c69"
+
+  depends_on :xcode => ["9.0", :build]
+
+  def install
+    # libxml2 has to be included in ISYSTEM_PATH for building one of
+    # dependencies. It didn't happen automatically before Xcode 9.3
+    # so homebrew patched environment variable to get it work.
+    # But since Xcode 9.3 includes it already, the build will fail
+    # because of redefinition of libxml2 module.
+    # It's a bug of homebrew but before it's fixed, it's easier
+    # to provide in-place workaround for now.
+    # Please remove this once homebrew is patched.
+
```

Comment:
Can we trim this empty line here and in `test do`? Thanks!

Reply:
Done in 8077f41f4b